Tests/Unit/Hook/FileDumpControllerHookTest.php

```diff
@@ -61,10 +61,11 @@ public function grantedUserCombinations()
         $feUser->user = [
             'uid' => '999',
         ];
-
+        $feUser->getKey('ses', 'Application')->willReturn('999');
```

Comment:
This should be done in the test instead of the data provider.

Reply:
Could you please elaborate what you mean by this? I understand you want me to remove this out of the data provider and change something in the test functions( _deniesAcces_ as well as the _grantsAccess_ ) ? Or am I not understanding you correctly?

Reply:
Yes, in the simplest case this line is moved to the tests which use the `grantedUserCombinations` data provider. You can run the tests locally to see if everything works.

Reply:
As discussed this perfectly makes sense here since it affects the test dummy object.